### PR TITLE
fixing the title found in the home page is done

### PR DESCRIPTION
**Pull request by Bethelb**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,92 +1,93 @@
 /**
  * App.js fro the main routing and basic layout
- * 
+ *
  * @author Practical IT
  */
-import React, { Fragment, useState } from 'react';
```
Comment (practicalit) [inline src/App.js:6]: Why do you change these values at all?

File: src/App.js [hunk 2]
```diff
@@ -1,92 +1,93 @@
...
   ];
 
   return (
     <Router>
-    <Fragment>
+      <Fragment>
+        <nav className="navbar navbar-light bg-light static-top">
+          <div className="container">
+            <a className="navbar-brand" href="#">
+              The NAME (Heritage) Project
```
Comment (practicalit) [inline src/App.js:32]: Probably this is the only change we want.

**Review by practicalit — COMMENTED** [review]
This is way too much change for a very single line task. I can't even see where the change is as there are tones of changes you have made. 

Please discard this branch and create a new one with new PR that reflects only the change requested.

Thank you.

**Comment by Bethelb** [comment]
deleting branch

**Pull request closed without merging**